src/client/components/evaluate/detail/CountySplit.tsx

```diff
@@ -0,0 +1,101 @@
+/** @jsx jsx */
+import { Box, Flex, jsx, ThemeUIStyleObject, Heading } from "theme-ui";
+import {
+  EvaluateMetricWithValue,
+  IProject,
+  IStaticMetadata,
+  RegionLookupProperties
+} from "../../../../shared/entities";
+import { useState, useEffect } from "react";
+import { getLabelLookup } from "../../map/labels";
+import { Resource } from "../../../resource";
+import { geoLevelLabel } from "../../../functions";
+
+const style: ThemeUIStyleObject = {
+  td: {
+    fontWeight: "body",
+    color: "gray.8",
+    fontSize: 1,
+    p: 2,
+    textAlign: "left",
+    verticalAlign: "bottom",
+    minWidth: "20px",
+    maxWidth: "200px",
+    mr: "10px"
+  },
+  fillBox: {
+    height: "10px",
+    width: "10px",
+    background: "#fed8b1",
+    opacity: "0.5",
+    outline: "1px solid black"
+  },
+  unfilledBox: {
+    height: "10px",
+    width: "10px",
+    background: "none",
+    outline: "1px solid black"
+  }
+};
+
+type CountyLookup = ReadonlyArray<string | undefined>;
+
+const CountySplitMetricDetail = ({
+  metric,
+  project,
+  staticMetadata,
+  regionProperties,
+  geoLevel
+}: {
+  readonly metric: EvaluateMetricWithValue;
+  readonly project?: IProject;
+  readonly geoLevel: string;
+  readonly regionProperties: Resource<readonly RegionLookupProperties[]>;
+  readonly staticMetadata?: IStaticMetadata;
+}) => {
+  const [countyLookup, setCountyLookup] = useState<CountyLookup | undefined>(undefined);
+
+  useEffect(() => {
+    if ("resource" in regionProperties && !countyLookup) {
+      // @ts-ignore
+      setCountyLookup(regionProperties.resource.map(c => c.name));
```

Comment:
We can appease the compiler here w/ some runtime type checking:
```suggestion
      setCountyLookup(regionProperties.resource.map(c => typeof c.name === "string" ? c.name : undefined));
```